README.md

```diff
@@ -0,0 +1,6 @@
+# teleop_twist_keyboard
+Generic Keyboard Teleop for ROS
+#Launch
+To launch run: `rosrun teleop_twist_keyboard teleop_twist_keyboard.py`
```

Comment:
`launch` in ROS has a very specific meaning (launch files and roslaunch). This should just read: "To run:"